### PR TITLE
Devel Pull Request

### PR DESCRIPTION
**Pull request by kbai**
Let me try again and if it fails I will send you the modification. Thank you.

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
The files changed require a buildbot test. Buildbot has started.

**Comment by buildbot-princeton** [comment]
Your changes passed the buildbot test. See [build 255](http://specfem3d-buildbot.princeton.edu/builders/specfem3d_builder/builds/255).

**Comment by komatits** [comment]
accept

On 03/02/2015 04:58 PM, buildbot-princeton wrote:
> Your changes passed the buildbot test. See build 255
> <http://specfem3d-buildbot.princeton.edu/builders/specfem3d_builder/builds/255>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem3d/pull/441#issuecomment-76737839>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr
